examples/all-clusters-app/nrfconnect/Kconfig

```diff
@@ -22,4 +22,6 @@ config STATE_LEDS
 	  Use LEDs to render the current state of the device such as the progress of commissioning of
 	  the device into a network or the factory reset initiation.
 
+rsource "../../../config/nrfconnect/chip-module/Kconfig.defaults"
```

Comment:
I think we have no other choice here, but when editing NCS samples you better use `source "${ZEPHYR_BASE}/../modules/...` construct to make sure that the sample will continue to work after being copied out-of-tree.